plugins/inputs/bind/bind_test.go

```diff
@@ -0,0 +1,200 @@
+package bind
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/influxdata/telegraf/testutil"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBindJsonStats(t *testing.T) {
+	ts := httptest.NewServer(http.FileServer(http.Dir("testdata")))
+	defer ts.Close()
+
+	b := Bind{
+		Urls: []string{ts.URL + "/json/v1"},
```

Comment:
I'm not sure I understand what you mean. I added a few memory contexts to the test data, but haven't written tests for them. As far as I remember, I didn't include any view data in the test files. I'm more or less doing this all in a purely theoretical sense now, as my requirements have moved on since I originally submitted this PR in 2016. I don't actually need this plugin in my current role, and am submitting it purely under the presumption that others may find it useful and continue work on it, once it is merged.